.github/workflows/nightwatch.yml

```
@@ -0,0 +1,68 @@
+name: dpc-sdp/ripple/nightwatch
+on:
+  workflow_dispatch:
+    inputs:
+      e2e:
+        type: boolean
+        required: true
+      e2e_project:
+        type: string
+        required: true
+      e2e_be_url:
+        type: string
+      e2e_fe_url:
+        type: string
+jobs:
+  e2e_be:
+    if: startsWith(github.ref_name , 'release/') && inputs.e2e
```

Hi @MarkCalvert please remove this release branch condition, it's not required anymore. Cheers!
```suggestion
    if: inputs.e2e
```